src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -43,6 +48,24 @@ public static Index parseIndex(String oneBasedIndex) throws IllegalValueExceptio
         }
         return Index.fromOneBased(Integer.parseInt(trimmedIndex));
     }
+    //@@ author arnollim
+    /**
+     * @param args into a String, which will be the filename which the .txt file will be saved as
+     * for e.g. if filename was the arg, then the file will be saved as filename.txt
+     * Leading and trailing whitespaces will be trimmed.
+     * @return "filename" for example.
+     * @throws IllegalValueException if there is no specified filepath.
+     */
+    public static String parseFilePath(String args) throws IllegalValueException {
+        final Matcher matcher = PRINT_ARGS_FORMAT.matcher(args.trim());
+        if (!matcher.matches()) {
+            throw new IllegalValueException(MESSAGE_INVALID_INDEX);
+        }
+        {
```

Comment:
what are these extra braces for? Travis not screaming at you?

Reply:
Fixed